ui/src/components/issuers.tsx/IssuerForm.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, Col, Divider, Form, Input, Row, Select } from "antd";
+import { useState } from "react";
+import IconBack from "src/assets/icons/arrow-narrow-left.svg?react";
+import { AuthBJJCredentialStatus, IssuerFormData, IssuerType } from "src/domain/identifier";
+import { VALUE_REQUIRED } from "src/utils/constants";
+
+export function IssuerForm({
+  onBack,
+  onSubmit,
+}: {
+  onBack: () => void;
+  onSubmit: (formValues: IssuerFormData) => void;
+}) {
+  const [form] = Form.useForm<IssuerFormData>();
+  const [showCredentialStatusField, setShowCredentialStatusField] = useState(false);
+
+  const handleFormChange = (changedValues: Partial<IssuerFormData>) => {
+    if (changedValues.type) {
+      if (changedValues.type === IssuerType.BJJ) {
+        setShowCredentialStatusField(true);
+      } else {
+        setShowCredentialStatusField(false);
+      }
+    }
```

Comment:
```
if (changedValues?.type === IssuerType.BJJ) {
        setShowCredentialStatusField(true);
      } else {
        setShowCredentialStatusField(false);
      }
```

Reply:
changedValues ​​cannot be undefined because this function is called when any field changes

Reply:
then do we need first if ` if (changedValues.type) {` ?